BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/security/service/MagicFolderUtil.java

```diff
@@ -42,6 +42,19 @@ void createGroupMagicFolder(long groupId, String magicKey) {
         createMagicFolder(groupId, magicKey, "b");
     }
 
+    public String getUserMagicFolderPath(long userId, String secret) {
+        return getMagicFolderPath(userId, secret, "a");
+    }
+
+    public String getGroupMagicFolderPath(long groupId, String secret) {
```

Comment:
I will suggest to change this metod and suer one to return `Path`  so latter you can use Path#resolve to get specifc sub folder or path.